tasks.go

```diff
@@ -14,9 +14,14 @@ import (
 // A Task performs some part of the RHMAP System Dump Tool.
 type Task func() error
 
+// Logger is a minimal interface implemented by log.Logger.
+type Logger interface {
+	Printf(format string, v ...interface{})
+}
+
 // RunAllDumpTasks runs all tasks known to the dump tool using concurrent workers.
 // Dump output goes to path.
-func RunAllDumpTasks(runner Runner, path string, workers int) {
+func RunAllDumpTasks(runner Runner, path string, workers int, fileOnlyLogger Logger) {
```

Comment:
I would like to introduce a "TaskRunner", it could then take a `logger` and a `fileOnlyLogger` as attributes.
Though for that would be nice to unify the code in this file first.